src/Raven.Studio/typescript/components/pages/resources/manageServer/storageReport/partials/StorageReportTable.tsx

```diff
@@ -0,0 +1,101 @@
+import { hasChildren, StorageReportItem } from "components/pages/resources/manageServer/storageReport/partials/common";
+import genUtils from "common/generalUtils";
+import React, { useId } from "react";
+import { withPreventDefault } from "components/utils/common";
+import { UncontrolledTooltip } from "reactstrap";
+import useUniqueId from "hooks/useUniqueId";
+
+interface StorageReportTableProps {
+    node: StorageReportItem;
+    root: StorageReportItem;
+    onNodeSelected: (node: StorageReportItem) => void;
```

Comment:
Same in other places
```suggestion
    handleNodeSelection: (node: StorageReportItem) => void;
```